readme.md

```diff
@@ -24,14 +24,14 @@ hexRgb('#4183c4');
 hexRgb('#fff');
 //=> {red: 255, green: 255, blue: 255, alpha: 255}
 
-hexRgb('#4183c488');
-//=> {red: 65, green: 131, blue: 196, alpha: 136}
+hexRgb('#22222299');
+//=> {red: 65, green: 131, blue: 196, alpha: 0.6}
 
-hexRgb('#0008');
-//=> {red: 0, green: 0, blue: 0, alpha: 136}
+hexRgb('#cd2222cc');
```

Comment:
This one was intentionally using the short format, so show that the short format also works with alpha.

Reply:
This one was removed due to your previous problem with the fraction digits.